### PR TITLE
Interactive list functionality

### PR DESCRIPTION
**Pull request by mahdimahdawi**
**This PR contain:**
- Implemented JavaScript modules to the project.
- Added event listener to checkbox for each list item and update the completed file as true.
- Implement functionality to item lists that are mark as completed.
- - Store the updating status to local storage.

### CODE REVIEW

File: src/list.js [hunk 1]
```diff
@@ -2,9 +2,10 @@
 import 'lodash';
```
Comment (vechicin) [inline src/list.js:2]: Please do not disable ESLint rules.

File: modules/makeComplete.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
```
Comment (vechicin) [inline modules/makeComplete.js:3]: Please do not disable ESLint rules.

File: modules/clearComplete.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```
Comment (vechicin) [inline modules/clearComplete.js:1]: Please do not disable ESLint rules.

File: src/list.js [hunk 4]
```diff
@@ -102,9 +108,12 @@ const displayTask = () => {
...
       e.addEventListener('click', editTask);
     });
-  } else {
-    document.querySelector('todo-list').innerHTML = '';
-  }
+    document.querySelectorAll('.check-box').forEach((e) => {
+      e.addEventListener('change', MakeComplete);
+    })
+  } 
+  // document.querySelector('todo-list').innerHTML = '';
```
Comment (vechicin) [inline src/list.js]: Please remove this line of commented code to keep a cleaner codebase.

File: src/list.js [hunk 5]
```diff
@@ -102,9 +108,12 @@ const displayTask = () => {
     document.querySelectorAll('.btn-edit').forEach((e) => {
       e.addEventListener('click', editTask);
     });
-  } else {
-    document.querySelector('todo-list').innerHTML = '';
-  }
+    document.querySelectorAll('.check-box').forEach((e) => {
+      e.addEventListener('change', MakeComplete);
+    })
```
Comment (vechicin) [inline src/list.js:116]: Please make sure you make the `completed` styling persist even after page refresh.

Screen Shot 2022-08-18 at 9 09 31

As you can see, the task is marked as `completed` but the styling doesn't reflect it. Please fix it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @mahdimahdawi ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing most of the required features for this milestone!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
**Fix Required Changes:**
- - After refresh the page the checkbox for completed task is stay check.
- Fixed linter Error.
- Clean dead code from project.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @mahdimahdawi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The PR title and description looks professional✔️
- No linter errors✔️
- You did well to make the changes requested by the previous reviewer✔️


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**